src/decorators/withCounter/withCounter.jsx

```diff
@@ -1,26 +1,38 @@
 // @flow
 import React, { type ComponentType } from 'react';
 import cx from 'classnames';
-import style from './style.scss';
+
+import styles from './style.scss';
 
 type Props = {
   value: string,
-  counterStyle: any,
-  maxLength: string,
+  counterClassName?: string,
```

Comment:
not sure if we need classname is a prop for this one

Reply:
I thought it might be necessary just in case, since we had `counterStyle` here.